src/features/high-fee-drawer/components/high-fee-confirmation.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useFormikContext } from 'formik';
+import { Button, Stack } from '@stacks/ui';
+
+import { useDrawers } from '@common/hooks/use-drawers';
+import { TransactionFormValues } from '@common/types';
+import { openInNewTab } from '@common/utils/open-in-new-tab';
+import { Link } from '@components/link';
+import { Caption, Title } from '@components/typography';
+
+const URL = 'https://hiro.so/questions/fee-estimates';
+
+export function HighFeeConfirmation(): JSX.Element | null {
+  const { handleSubmit, values } = useFormikContext<TransactionFormValues>();
+  const { showHighFeeConfirmation, setShowHighFeeConfirmation } = useDrawers();
+
+  return (
+    <>
+      <Title fontSize="20px" fontWeight={400} lineHeight="28px">
+        Are you sure you want to pay {values.txFee} STX in fees for this transaction?
+      </Title>
+      <Caption>
+        This action cannot be undone and the fees won't be returned, even if the transaction fails.{' '}
+        <Link fontSize="14px" onClick={() => openInNewTab(URL)}>
+          Learn more
+        </Link>
+      </Caption>
+      <Stack isInline mt="loose">
+        <Button
+          borderRadius="10px"
+          mode="tertiary"
+          onClick={() => setShowHighFeeConfirmation(!showHighFeeConfirmation)}
+          width="50%"
+        >
+          Edit fee
+        </Button>
+        <Button
+          borderRadius="10px"
```

Comment:
Do we need to add the border radius here? 

I gather the designs may have more rounded buttons than the UI library defaults to, but it'll get messy if we try to apply these in _all_ instances of buttons henceforth.

Reply:
I just copied these from the `edit-nonce-drawer` to make sure they matched. I'll check though, I am sure we do need to add them to adjust according to the design as needed. The border radius does change I've noticed in Figma ...definitely depending on the size of the button we are using.